src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabViewItem.cs

```diff
@@ -0,0 +1,418 @@
+﻿using System.Numerics;
+using Microsoft.UI.Xaml.Automation.Peers;
+using Uno.UI.Helpers.WinUI;
+using Windows.Devices.Input;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Core;
+using Windows.UI.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class TabViewItem : ListViewItem
+	{
+		private const string c_overlayCornerRadiusKey = "OverlayCornerRadius";
+		private const string SR_TabViewCloseButtonName = "TabViewCloseButtonName";
+
+		private bool m_firstTimeSettingToolTip = true;
+		private bool m_hasPointerCapture = false;
+		private bool m_isMiddlePointerButtonPressed = false;
+		private bool m_isDragging = false;
+		private bool m_isPointerOver = false;
+		private TabViewCloseButtonOverlayMode m_closeButtonOverlayMode = TabViewCloseButtonOverlayMode.Auto;
+		private TabViewWidthMode m_tabViewWidthMode = TabViewWidthMode.Equal;
+		private Button m_closeButton;
+		private ToolTip m_toolTip;
+		private object m_shadow;
+		private TabView m_parentTabView;
+
+		public TabViewItem()
+		{
+			//__RP_Marker_ClassById(RuntimeProfiler.ProfId_TabViewItem);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6063676)